librz/core/cmd_analysis.c

```diff
@@ -8088,6 +8088,115 @@ RZ_IPI RzCmdStatus rz_analysis_rtti_demangle_class_name_handler(RzCore *core, in
 	return RZ_CMD_STATUS_ERROR;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_print_global_variable_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	char *var_name = NULL;
+	if (argc == 2) {
+		var_name = (char *)argv[1];
+	}
+	rz_analysis_var_global_list_show(core->analysis, state, var_name);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analysis_global_variable_add_handler(RzCore *core, int argc, const char **argv) {
+	if (argc < 4 || argc > 5) {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	char *var_name = (char *)argv[1];
+	ut64 addr = rz_num_math(core->num, (char *)argv[2]);
+	char *type = (char *)argv[3];
+	char *comment = NULL;
+	if (argc == 5) {
+		comment = (char *)argv[4];
+	}
+
+	RzTypeParser *parser = rz_type_parser_new();
+	if (!parser) {
+		return RZ_CMD_STATUS_ERROR;
+	}
+	char *errmsg = NULL;
+	RzType *typ = rz_type_parse_string_single(parser, type, &errmsg);
+	if (errmsg) {
+		RZ_LOG_ERROR("%s : Error parsing type: \"%s\" message:\n%s\n", __FUNCTION__, type, errmsg);
+		free(errmsg);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	RzAnalysisVarGlobal *glob = rz_analysis_var_global_new(var_name, addr, comment);
+	if (!glob) {
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_analysis_var_global_set_type(glob, typ);
+	if (rz_analysis_var_global_add(core->analysis, glob)) {
```

Comment:
please do the opposite
```suggestion
	if (!rz_analysis_var_global_add(core->analysis, glob)) {
	    ...
```

For consistency. All the indentend code in the function is "error code", so this looks a bit out of place.